layouts/shortcodes/observability_pipelines/configure_existing_pipelines/install_worker/linux_apt.md

```diff
@@ -1,9 +1,8 @@
 
 1. Click **Select API key** to choose the Datadog API key you want to use.
-1. Run the one-step command provided in the UI to re-install the Worker:
-    ```shell
-    DD_API_KEY=<DATADOG_API_KEY> DD_OP_PIPELINE_ID=<PIPELINES_ID> DD_SITE=<DATADOG_SITE> <SOURCE_ENV_VARIABLES> <DESTINATION_ENV_VARIABLES> bash -c "$(curl -L https://install.datadoghq.com/scripts/install_script_op_worker2.sh)"
-    ```
+1. Run the one-step command provided in the UI to re-install the Worker.
+
+    **Note**:  The environment variables used by the Worker in `/etc/default/observability-pipelines-worker` are not updated on subsequent runs of the install script.  If changes are needed, update the file manually and restart the Worker.
```

Comment:
Hi @sghall I was wondering if we should just remove this section for the Update Existing Pipelines doc, since it's in the specific [Update source and destination environment](https://docs-staging.datadoghq.com/may/add-note-one-line-install/observability_pipelines/update_existing_pipelines/?tab=datadogagent#update-source-or-destination-variables) section, which means:
1) They already have the Worker installed.
2) They need to update that file to update the environment variables, so there's no point in running the one-line command.

Reply:
Chatted over zoom. Let's leave it.